### DISCUSSION TITLE
[Script request] telerising-api

### DISCUSSION THREAD

**Discussion by CustomerJFK** [community-scripts/ProxmoxVE#913] [Request script]
### Application Name

telerising-api

### Website

https://github.com/sunsettrack4/telerising-api

### Description

Hello 
Can anyone be so kind to create a Proxmox LXC script for telerising?

Thanks

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Add description for this and for what you doing. We discuss this after next Project Pipeline Workload

**Comment by CustomerJFK** [comment]
Hi 
It's a self-hosted TV-Server. 
(API web application providing Zattoo TV streams and much more...)
Thanks!
